python/lsst/daf/butler/core/config.py

```diff
@@ -126,6 +126,7 @@ class Config(collections.abc.MutableMapping):
     Note that adding a multi-level key implicitly creates any nesting levels
     that do not exist, but removing multi-level keys does not automatically
     remove empty nesting levels.  As a result::
+
         >>> c = Config()
```

Comment:
The `::` and `>>>` syntaxes are (I believe) mutually exclusive. I'd do:

```rst
As a result:

>>> c = Config()
```

and so on.